integrations/pytorch/vision.py

```diff
@@ -1126,6 +1137,7 @@ def train(args, model, train_loader, val_loader, input_shape, save_dir, loggers)
         # initial baseline eval run
         tester.run_epoch(val_loader, epoch=epoch - 1, max_steps=args.debug_steps)
 
+    _save_recipe(recipe_manager=manager, save_dir=save_dir)
```

Comment:
this should be moved to the if statement below since the manager is set to None if we are in `eval_mode`

Reply:
Good catch!